models/game.rb

```diff
@@ -103,7 +103,9 @@ def ordered_players
 
   def to_h(include_actions: false, player: nil)
     actions_h = include_actions ? actions.map(&:to_h) : []
-    settings_h = include_actions ? settings.to_h : {}
+    settings_h = settings.to_h
+
+    settings_h.delete('seed') unless include_actions
```

Comment:
why do we delete seed? just keep it in. once you make this change go ahead and merge

Reply:
That was the one thing I saw that was really being excluded by the old logic of only including settings when actions were also included. Will make this change when I get back to a computer...